documentation/assemblies/cruise-control/assembly-cruise-control-concepts.adoc

```diff
@@ -3,11 +3,16 @@
 // master.adoc
 
 [id='cruise-control-concepts-{context}']
-= Cruise Control
+= Cruise Control for cluster rebalancing
 
-You can deploy {CruiseControlProject} and access a subset of its features through {ProductName} custom resources.
+You can deploy {CruiseControlProject} to your {ProductName} cluster and access its _cluster rebalancing_ feature through custom resources.
 
-Example YAML files for deploying Cruise Control and setting optimization goals are provided in `examples/cruise-control/`.
+Cruise Control is an open source system for automating Kafka operations, including monitoring cluster workload, rebalancing a cluster based on predefined constraints, and detecting and fixing anomalies, among others. 
+It consists of four main components--the Load Monitor, the Analyzer, the Anomaly Detector, and the Executor--and a REST API for all interactions.
+{ProductName} utilizes the REST API to support the cluster rebalancing features of Cruise Control.
+Other features, such as anomaly detection, Slack notifications, and changing the topic replication factor, are not supported.
```

Comment:
Could we change this to "not currently supported"? We will most likely be add topic replication in the future, not sure about slack notifications though? Maybe change slack notifications to something ore general like "anomaly notifications"